clients/admin-ui/src/features/plus/plus.slice.ts

```diff
@@ -167,7 +167,7 @@ const plusApi = baseApi.injectEndpoints({
         url: `plus/custom-metadata/custom-field/${id}`,
         method: "DELETE",
       }),
-      invalidatesTags: ["Custom Fields"],
+      invalidatesTags: ["Custom Fields", "Datamap"],
```

Comment:
ah, so glad the refactor is paying off already 🙌 just too bad it wasn't in time for the hot fix!